docs/getting-started-guides/locally.md

```diff
@@ -80,6 +80,9 @@ cd kubernetes
 hack/local-up-cluster.sh
 ```
 
+Note there are ways that you can "hack" the above script further, to test different permutations of Kubernetes options.
+For an example of using hack/local-up-cluster.sh to test high availability apiserver configurations, see the bottom of this doc.
```

Comment:
Recommend removing these two lines, they're no longer accurate--nothing about this has to do with HA.